crates/common/src/signature.rs

```diff
@@ -76,17 +75,35 @@ pub fn sign_builder_root(
     sign_message(secret_key, &signing_root)
 }
 
+pub fn sign_commit_boost_root(
+    chain: Chain,
+    secret_key: &BlsSecretKey,
+    object_root: [u8; 32],
+) -> BlsSignature {
+    let domain = compute_domain(chain, COMMIT_BOOST_DOMAIN);
+    let signing_root = compute_signing_root(object_root, domain);
```

Comment:
Maybe unite those two lines in a private `sign_for_domain(domain: [u8; 32], secret_key: &BlsSecretKey, object_root: [u8; 32])` which will be called with a specified domain from `sign_builder_root` and `signer_commit_boost_root`?